FAIMS/FAIMS3
# fix: package rollback

## JIRA Ticket

[BSS-334](https://jira.csiro.au/browse/BSS-334)

## Description

Issue in modified `package.json` files caused app to not load once packages purged.

## Proposed Changes

Reverted updates to packages

## How to Test

1. `/scripts/purge.sh .`
2. `npm install`
3. `npm run start-app`
4. Verify the app runs in your browser

## Additional Information

## Checklist
